CODEOWNERS

```
@@ -85,7 +85,7 @@ NOTICES     @garyverhaegen-da @aherrmann-da @cocreature @dasormeter
 /navigator/backend/src/main/scala/com/digitalasset/navigator/json/    @S11001001
 
 # Misc
-/docs/              @bame-da @nemanja-da
+/docs/              @bame-da @nemanja-da @cocreature
```

```suggestion
/docs/              @cocreature @stefanobaghino-da 
```
1. I would add myself as a backup if you don't mind
2. If Bernhard and Nemanja are not looking at `/docs/` let's acknowledge that.

---

> I would add myself as a backup if you don't mind
not at all, thanks for offering